celery_prometheus_exporter.py

```diff
@@ -130,6 +131,9 @@ def main():
         '--broker', dest='broker', default=DEFAULT_BROKER,
         help="URL to the Celery broker. Defaults to {}".format(DEFAULT_BROKER))
     parser.add_argument(
+        '--transport_options', dest='transport_options',
```

Comment:
I'm not completely sure, but would perhaps a dash instead of an underscore in the argument name (`--transport-options` instead of `--transport_options`) be better?